symfony/client/src/orangehrmAuthenticationPlugin/pages/ResetPassword.vue

```diff
@@ -112,7 +112,7 @@ export default {
         confirmPassword: [
           required,
           shouldNotExceedCharLength(64),
-          v => (!!v && v === this.password) || 'Passwords do not match',
+          v => (!!v && v === this.newPassword) || 'Passwords do not match',
```

Comment:
does `this.newPassword` work on your end? when trying on my local it doesn't work and always says Passwords do not match. Shouldn't it be
```suggestion
          v => (!!v && v === this.user.newPassword) || 'Passwords do not match',
```